pkg/juicefs/juicefs.go

```diff
@@ -318,6 +318,10 @@ func (j *juicefs) JfsMount(ctx context.Context, volumeID string, target string,
 	if err := j.validTarget(target); err != nil {
 		return nil, err
 	}
+
+	if err := j.overwriteVolCtxWithPVCAnnotations(ctx, volumeID, volCtx); err != nil {
+		klog.Warningf("Overwrite volCtx with PVC annotations error: %v", err)
```

Comment:
maybe use a log level of error since this will definitely go against user's expectation?